index.html

```diff
@@ -3,6 +3,19 @@
 title: Bazel - a fast, scalable, multi-language and extensible build system"
 ---
 <div class="home">
+  <div class="bazelcon">
+    <div class="container">
+      <div class="row">
+        <div class="col-sm-12">
+          <p class="text-center">
+            <b>BazelCon 2020</b> is happening virtually this year: mark your
+            calendar for 12-13 November! <a
```

Comment:
<b>BazelCon 2020</b> is virtual this year. Save the date
            for November 12–13, then <a
            href="http://goo.gle/bazelcon">register and view the agenda</a>!

(Removes "here" and makes it a bit shorter to prevent orphaned words on less than full screen.

Reply:
Done.